below/model/src/cgroup.rs

```diff
@@ -711,6 +717,42 @@ impl CgroupMemoryModel {
     }
 }
 
+#[derive(
+    Clone,
+    Debug,
+    Default,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    below_derive::Queriable
+)]
+pub struct CgroupPidsModel {
+    pub pids_current: Option<u64>,
```

Comment:
Let's call this tids_current to avoid any confusion that these are process IDs rather than thread IDs

Reply:
I have to disagree here, the pids controller has no relation to threads.

Reply:
> Note that PIDs used in this controller refer to TIDs, process IDs as used by the kernel.

You can verify yourself by running single process (multi threaded) service in its own cgroup and checking pids.current.

Reply:
but IMO it's confusing the users with the naming. If it's the case, then no problem, I will replace ✌️

Reply:
Updated, thanks for the review!